scripts/composer/ScriptHandler.php

```diff
@@ -47,8 +47,8 @@ public static function createRequiredFiles(Event $event) {
       require_once $drupalRoot . '/core/includes/bootstrap.inc';
       require_once $drupalRoot . '/core/includes/install.inc';
       $settings['config_directories'] = [
-        CONFIG_SYNC_DIRECTORY => (object) [
-          'value' => Path::makeRelative($drupalFinder->getComposerRoot() . '/config/sync', $drupalRoot),
+        'config_sync_directory' => (object) [
```

Comment:
Not sure what is needed here. This change makes the error go away, and I'm sure I saw it suggested somewhere, but note too https://github.com/UN-OCHA/iasc8/pull/504/files#diff-1006f32e007da05cefdc1061588471f49e9f2916d1a66b340b6b2510b4375898
and https://www.drupal.org/node/3018145

Reply:
We use `root . '/config'` for all projects, so in *theory* that should work.